tools/android/MASTG-TOOL-0015.md

```diff
@@ -1,76 +1,158 @@
-$ dz> run app.package.list
+### Example usage
 
-# Find the package name of a specific app
-$ dz> run app.package.list -f (string to be searched)
+Once set up, you can use drozer to perform reconnaissance and exploitation of Android applications from the perspective of a malicious app on the device. [The drozer User Manual](https://labs.withsecure.com/tools/drozer#3 "drozer User Manual") introduces an intentionally vulnerable application - [sieve](https://github.com/WithSecureLabs/sieve "GitHub repo - sieve") - together with step-by-step exploitation instructions.
 
-# See basic information
-$ dz> run app.package.info -a (package name)
+Some common drozer commands include:
 
-# Identify the exported application components
-$ dz> run app.package.attacksurface (package name)
+#### Searching for applications on the device:
+```
+run app.package.list -f <keyword>
+```
+This lists basic informations about any packages containing the word "<keyword>" in their bundle identifier. This includes package names, key directories used by the application, and any permissions used or defined by the application.
 
-# Identify the list of exported Activities
-$ dz> run app.activity.info -a (package name)
+#### Enumerating the attack surface of an app:
+```
+run app.package.attacksurface <package>
+```
+This command inspects the target app's manifest and provides a report on any exported components of the application, and verifies whether the application is debuggable.
 
-# Launch the exported Activities
-$ dz> run app.activity.start --component (package name) (component name)
+Once the attack surface has been identified, you can obtain more specific information about each component class. For example, to list  Activities, you can use the following command:
+```
+run app.activity.info -a <package>
+```
+This lists the names of all exported Activities, together with the permissions required to interact with them.
 
-# Identify the list of exported Broadcast receivers
-$ dz> run app.broadcast.info -a (package name)
+#### Starting an Activity
+In order to launch an exported activity, use the following command:
+```
+run app.activity.start --component <package> <component name>
+```
 
-# Send a message to a Broadcast receiver
-$ dz> run app.broadcast.send --action (broadcast receiver name) -- extra (number of arguments)
+When calling `app.activity.start`, you can build a much more complex intent. As with all drozer modules, you can request more usage information:
 
-# Detect SQL injections in content providers
-$ dz> run scanner.provider.injection -a (package name)
 ```
+dz> help app.activity.start
+Attempting to run shell module
+usage: run app.activity.start [-h] [--action ACTION] [--category CATEGORY [CATEGORY ...]] [--component PACKAGE COMPONENT] [--data-uri DATA_URI] [--extra TYPE KEY VALUE] [--flags FLAGS [FLAGS ...]] [--mimetype MIMETYPE]
+```
+
+You can learn more about how intents are created by running `help intents`
+
+#### Further information
```

Comment:
This link is already used in "### Example usage". Can you remove this short section and merge it with "### Example usage"?